Sources/TransifexNative/Core.swift

```diff
@@ -23,6 +23,64 @@ public enum TXRenderingStategy : Int {
     case icu
 }
 
+/// Bypasses swizzling and offers the bundled translation for the passed locale, if found.
+///
+/// If the string is not found the logic returns the value of the `params` dictionary and if this is
+/// also not found, it returns the `sourceString`.
```

Comment:
It feels like this could/should be even more descriptive. I try to imagine myself bumping into this code 1 year from now and I don't think I'll remember the exact case this refers to. Maybe mention something about **when** this case appears (i.e. when those key=value files are used).

Reply:
I have updated the docstring there. I have also updated the `README.md` file and pushed the changes so have a look and let me know :)